build.sbt

```diff
@@ -72,7 +72,9 @@ lazy val lambda = (project in file("lambda"))
       name := "tdr-database-migration-lambda",
       libraryDependencies ++= Seq(
         "org.flywaydb" % "flyway-core" % "6.1.4",
-	      "org.postgresql" % "postgresql" % "42.2.11"
+        "software.amazon.awssdk" % "rds" % "2.16.16",
+	      "org.postgresql" % "postgresql" % "42.2.11",
```

Comment:
Not a big deal, but the indentation is off.

Reply:
I must have opened it in vim at some point. It likes to pick something random for the indentation.